hudi-common/src/main/java/org/apache/hudi/common/table/TableSchemaResolver.java

```diff
@@ -353,6 +361,56 @@ public static boolean isSchemaCompatible(String oldSchema, String newSchema) {
     return isSchemaCompatible(new Schema.Parser().parse(oldSchema), new Schema.Parser().parse(newSchema));
   }
 
+  /**
+   * Check if both schemas are equivalent.
+   * @param oldSchema old schema to be validated against.
+   * @param newSchema new schema to be validated against.
+   * @return {@code true} if both schema matches. else {@code false}.
+   */
+  public static boolean isSchemaEquals(Schema oldSchema, Schema newSchema) {
```

Comment:
Instead of these custom methods, can you implement the Comparator for Schema. The comparator contract for "0" meaning equal and non-zero providing which one is larger should suffice.

Reply:
sure. let me see.